portfolio/src/main/webapp/script.js

```diff
@@ -15,6 +15,7 @@
 /**
  * Adds a random greeting to the page.
```

Comment:
Update this comment, since you're generating random quotes now instead of greetings